### PR TITLE
Implement on Taskmanagement make apis and models debug in custom base…

### PR DESCRIPTION
**Pull request by harmeshverma01**
… user

### CODE REVIEW

File: Task_management/urls.py [hunk 1]
```diff
@@ -14,8 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('user.urls')),
+    path('', include('task.urls'))
```
Comment (akashSharma135) [inline Task_management/urls.py:22]: make correct url patterns

File: user/views.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+from .utils import admin_required
+from .models import User
+# Create your views here.
+
+class Userview(APIView):
+    serializer_class = UserSerializer
+    permission_classes = [permissions.IsAuthenticated, admin_required]
+    authentication_classes = [authentication.TokenAuthentication]
+    
+    def get(self, request, id=None):
```
Comment (akashSharma135) [inline user/views.py:19]: Always use pagination for large data.

File: user/views.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        except:
+            return Response(({'message' : 'User does not found'}), status=status.HTTP_204_NO_CONTENT)
+        
+    def delete(self, request, id=None):
+        user = User.objects.get(id=id)
+        user.delete()
+        return Response(({'message': 'user is deleted successfully'}), status=status.HTTP_204_NO_CONTENT)
+
+
+class ManagerView(APIView):
```
Comment (akashSharma135) [inline user/views.py:69]: What's the need of this API, It can be in get users API

Comment (harmeshverma01) [reply user/views.py:69]: fixed

File: user/manager.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    def create_superuser(self, email, password):
+        user = self.create_user(
+            email,
+            password=make_password(password)
+        )
+        user.is_staff = True
+        user.is_admin = True
+        user.is_superuser = True
+        user.save(using=self._db)
+        print(user.__dict__)
```
[the remark this answers is not in the document]
Comment (harmeshverma01) [reply user/manager.py:34]: fixed

File: Task_management/urls.py (hunk 1 above)
Comment (harmeshverma01) [reply Task_management/urls.py:22]: fixed

File: user/views.py (hunk 2 above)
Comment (harmeshverma01) [reply user/views.py:19]: fixed

**Pull request merged**